scripts/ifAlias

```diff
@@ -18,8 +18,17 @@ BASE='.1.3.6.1.2.1.31.1.1.1.18'
 GET_TYPE="$1"
 GET_OID="$2"
 
+UNAME="$(/usr/bin/uname)"
+
+COUNT="$(echo "${UNAME}" | grep -ci 'bsd$')"
+if [ "${COUNT}" -eq 1 ]; then
```

Comment:
`if [ "$(echo "${UNAME}" | grep -ci 'bsd$')"  -eq 1 ] ; then`  We could thus do without a global `COUNT` variable. It's reassigned as local var in `alias_from_ip_link`